solutions/usaco-988.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: usaco-988
+source: USACO Bronze 2020 January
+title: Photoshoot
+author: Ananth Kashyap
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_photo_bronze_jan20.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<PySection>
+
+```py
+data = open('photo.in').read().strip().split('\n')
+n = int(data[0])
+b = list(map(int, data[1].split(' ')))
+
+a = []
```

Comment:
more descriptive variable names :)

Reply:
you can keep the `n` but at least rename `a` and `b` they aren't explicit at all

Reply:
they're technically from the problem :stuck_out_tongue:

Reply:
don't think anyone reads the series, please just change those